flixel/system/frontEnds/BitmapFrontEnd.hx

```diff
@@ -302,7 +303,7 @@ class BitmapFrontEnd
 
 	/**
 	 * Totally removes FlxGraphic object with specified key.
-	 * @param	key	the key for cached FlxGraphic object.
+	 * @param key Key of the cached FlxGraphic object.
```

Comment:
```suggestion
	 * @param   key  Key of the cached graphic.
```

Calling it a "FlxGraphic object" is wordy and requires backticks, lets just say "graphic"